test/lib/absinthe/extensions_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Absinthe.ExtensionsTest do
+  use Absinthe.Case, async: true
```

Comment:
@fishcakez this module serves as a very minimal sketch of what setting extensions looks like.